src/scss/includes/panels/categories.scss

```diff
@@ -38,14 +38,6 @@
 
 .category__panel__error {
   padding: 30px 40px 50px;
```

Comment:
If you want to go on with this change for visual consistency, let's set the bottom padding to the same value as the other panel…